Description: Fix -Wdeprecated-declarations compiler warning.
 In particular, the following warning:
 gui.c: In function 'show_message':
 gui.c:444:2: warning: 'gtk_misc_set_padding' is deprecated
 (declared at /usr/include/gtk-3.0/gtk/deprecated/gtkmisc.h:80) [-Wdeprecated-declarations]
   gtk_misc_set_padding(&GTK_LABEL(label)->misc, 10, 10);
   ^
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2014-11-23

--- a/gui.c
+++ b/gui.c
@@ -441,7 +441,10 @@
 	}
 
 	/* add the label, and show everything we've added to the dialog. */
-	gtk_misc_set_padding(&GTK_LABEL(label)->misc, 10, 10);
+	gtk_widget_set_margin_start(label, 10);
+	gtk_widget_set_margin_end(label, 10);
+	gtk_widget_set_margin_top(label, 10);
+	gtk_widget_set_margin_bottom(label, 10);
 	gtk_container_add(GTK_CONTAINER(gtk_dialog_get_content_area(GTK_DIALOG(dialog))), label);
 	gtk_widget_show_all(dialog);
 
